pkg/v1/tkr/Dockerfile

```diff
@@ -15,6 +15,7 @@ COPY go.mod go.mod
 COPY go.sum go.sum
 COPY apis/ apis/
 COPY cli/runtime cli/runtime
+COPY packageclients/ packageclients/
```

Comment:
We discussed offline that is an interim step required for tkr components to be able to build. Should go away when tkr components are refactored to their own go mod.